src/config.ts

```diff
@@ -159,6 +159,7 @@ const EXCLUDED_SOURCES = (() => {
 
 const gasSchedule: { [key in ERC20BridgeSource]: number } = {
     [ERC20BridgeSource.Native]: 1.5e5,
+    [ERC20BridgeSource.Rfqt]: 1.5e5,
```

Comment:
I totally could be missing something, but is this still relevant? Also see https://github.com/0xProject/0x-monorepo/pull/2555/files#r410517580

I expected Rfqt orders just to be using the ERC20BridgeSource.Native bridge source.

Reply:
Usage of this is for a few reasons:
1) Exclude certain sources (controlled via API params) _[Would you disable Native but not RFQT?]_
2) Track internally in asset-swapper to avoid conflicting sources _[There's no conflicts with 0x orders]_
3) Calculate cost based off the source _[this is the same cost as Native]_

If we don't see ourselves doing the above, or if it is coupled to `Native`, then there may not be a need for it?

With the 0xAPI we'd definitely want to consider merging in `Rfqt` and `Native` to the composition source breakdown and show it as `0x`.

Reply:
I took this out for now, in https://github.com/0xProject/0x-api/pull/172/commits/1d9ad66b0f6720442eab037a1144c4027f66eca4.  See also https://github.com/0xProject/0x-monorepo/pull/2555#discussion_r410517580

I'm up for using this for something cool, but I'd rather not do it in this PR.